### PR TITLE
feat(next): tweak i18n

### PR DESCRIPTION
**Pull request by florian-lefebvre**
#### Description (required)

Puts `locales` before `defaultLocale`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/en/guides/internationalization.mdx [hunk 1]
```diff
@@ -22,14 +22,14 @@ You can also choose to [add your own i18n logic in addition to or instead of Ast
 
 ## Configure i18n routing
 
-Both a default language ([`defaultLocale`](/en/reference/configuration-reference/#i18ndefaultlocale)) and a list of all supported languages ([`locales`](/en/reference/configuration-reference/#i18nlocales)) must be specified in an `i18n` configuration object. Additionally, you can configure more specific routing and fallback behavior to match your desired URLs.
+Both a list of all supported languages ([`locales`](/en/reference/configuration-reference/#i18nlocales)) and a default language ([`defaultLocale`](/en/reference/configuration-reference/#i18ndefaultlocale), must be one of the supported locales specified in `locales`) must be specified in an `i18n` configuration object. Additionally, you can configure more specific routing and fallback behavior to match your desired URLs.
```
Comment (casungo) [inline src/content/docs/en/guides/internationalization.mdx:25]:
```suggestion
Both a list of all supported languages ([`locales`](/en/reference/configuration-reference/#i18nlocales)) and a default language ([`defaultLocale`](/en/reference/configuration-reference/#i18ndefaultlocale)), which must be one of the languages listed in `locales`, need to be specified in an `i18n` configuration object. Additionally, you can configure more specific routing and fallback behavior to match your desired URLs.
```

Previously, it sounded a bit off to me. Tried to fix it

**Review by casungo — CHANGES_REQUESTED** [review]
just a minor tweak :)

**Review by casungo — APPROVED** [review]
LGTM!

**Review by sarah11918 — APPROVED** [review]
LGTM! Thanks @florian-lefebvre !

**Pull request merged**